src/index.ts

```diff
@@ -78,3 +76,9 @@ function sendViolationBySeverity(msg: MarkdownString, fileName: string, line: nu
       break
   }
 }
+
+let defaultViolationFormatter: ViolationFormatter = {
+  format: (violation: Violation): string => {
+    return violation.message
+  }
+}
```

Comment:
Default formatter to mirror the current behavior in case it's not set in `CheckstyleConfig` (should rename that interface at some point 👀 )